src/components/Admin/index.jsx

```diff
@@ -36,46 +37,64 @@ class Admin extends React.Component {
     const defaultData = {
       title: 'Full Report',
       component: <EnrollmentsTable />,
+      csvFetchMethod: 'downloadCsv',
```

Comment:
Is the thought this would be the endpoint such as `EnterpriseApiDataService.fetchCourseEnrollmentsCsv`?  I think that would make the most sense, and if so I'd probably remove the `csvOptions` argument and just create multiple methods in the fetchCourseEnrollmentsCsv.

It will mean the top of this file will have a bunch of imports:
```
import {
  fetchCourseEnrollmentsCsv,
  fetchLearnersNotInCourseCsv,
  fetchLearnersEnrolledCsv,
....
} from '../../data/services/EnterpriseApiDataService';

This would keep all of the "logic" around the endpoints in the data service layer.